### ISSUE TITLE
[Bug] Jormungandr spawns  without tail if AoE dmg is present

### ISSUE THREAD

**Issue by TaipionsSalemClient** [PavelZinchenko/event-horizon-frontier-builds#382]
Current behaviour:
The jorms tail **regeneration** is interrupted when it takes damage, that's ok.
But the tail does not **spawn** when the ship spawns if there is AoE damage present (or it is otherwise damaged).

Expected:
When (re-)spawning, jorm spawns with a full tail regardless of AoE damage being present or not.
(Tail regeneration should still be interrupted through damage as it is.)

**Comment by SuplexCity25284** [comment]
Yeah, that makes sense.  It should fully spawn, and, then be damaged and have the tail take damage, not spawn with the damage already done.
